static/css/global.css

```diff
@@ -0,0 +1,65 @@
+.table-container {
```

Comment:
for understanding - can you explain to me what part each section represents in the screenshot?
thanks!

@shlior7